ui/admin/app/templates/scopes/scope/host-catalogs/host-catalog/host-sets/host-set/hosts/index.hbs

```diff
@@ -28,82 +28,67 @@
-            {{/if}}
-          </header.row>
-        </table.header>
+      <Hds::Table
+        @valign='middle'
+        @model={{@model.hosts}}
+        @columns={{this.columns}}
+      >
 
-        <table.body as |body|>
-          {{#each @model.hosts as |host|}}
-            <body.row as |row|>
-              <row.headerCell>
-                {{#if host.isStatic}}
-                  <LinkTo
-                    @route='scopes.scope.host-catalogs.host-catalog.hosts.host'
-                    @model={{host.id}}
-                  >
-                    {{host.displayName}}
-                  </LinkTo>
-                {{else}}
-                  {{! Link to a different subroute for dynamic/plugin hosts }}
-                  <LinkTo
-                    @route='scopes.scope.host-catalogs.host-catalog.host-sets.host-set.hosts.host'
-                    @model={{host.id}}
-                  >
-                    {{host.displayName}}
-                  </LinkTo>
-                {{/if}}
-                <p>{{host.description}}</p>
-              </row.headerCell>
-              <row.cell>
-                <HostCatalogTypeBadge @model={{host}} />
-              </row.cell>
-              <row.cell>
-                <Copyable
-                  @text={{host.id}}
-                  @buttonText={{t 'actions.copy-to-clipboard'}}
-                  @acknowledgeText={{t 'states.copied'}}
+        <:body as |B|>
+          <B.Tr>
+            <B.Td>
+              {{#if B.data.isStatic}}
+                <LinkTo
+                  @route='scopes.scope.host-catalogs.host-catalog.hosts.host'
+                  @model={{B.data.id}}
+                >
+                  {{B.data.displayName}}
+                </LinkTo>
+              {{else}}
+                {{! Link to a different subroute for dynamic/plugin hosts }}
+                <LinkTo
+                  @route='scopes.scope.host-catalogs.host-catalog.host-sets.host-set.hosts.host'
+                  @model={{B.data.id}}
                 >
-                  <code>{{host.id}}</code>
-                </Copyable>
-              </row.cell>
-              <row.cell>
-                {{#if (can 'removeHosts hostSet' @model.hostSet)}}
-                  <Rose::Dropdown
-                    @icon='flight-icons/svg/more-horizontal-16'
-                    @iconOnly={{true}}
-                    @showCaret={{false}}
-                    @dropdownRight={{true}}
+                  {{B.data.displayName}}
+                </LinkTo>
+              {{/if}}
+              <Hds::Text::Body @tag='p'>{{B.data.description}}</Hds::Text::Body>
+            </B.Td>
+            <B.Td>
+              <HostCatalogTypeBadge @model={{B.data}} />
+            </B.Td>
+            <B.Td>
+              <Hds::Copy::Snippet
+                @textToCopy={{B.data.id}}
+                @color='secondary'
+              />
+            </B.Td>
+            {{#if (can 'removeHosts hostSet' @model.hostSet)}}
```

Comment:
The conditional, should be nested inside the `<td>` and not outside, otherwise table rows are going to be different, table should be consistent.

[Here an example](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/app/templates/scopes/scope/groups/group/members.hbs#L68-L86) of this exact use case within other template we already migrate to HDS table

Reply:
@calcaide I moved this condition outside so that it doesn't have to render that `td`.  If I move it back outside the `td` will still render with nothing in it and I will have to provide the `dynamic` table a blank header label.  Which I did in this last [commit](https://github.com/hashicorp/boundary-ui/pull/2049/files#diff-1e0487df0f54bc897aae9166eefaf436cd6c19af19b68e6200bde0f2a6f398c2R39).

Reply:
@dvonanderson 
So actions column is JUST available for `static` types? so `dynamic` types have no actions?

Reply:
@DhariniJeeva correct me if I am wrong, but the `host-set` containing dynamic host can be deleted, but not the dynamic `host` itself?

Reply:
Unfortunately, I do not recall. Have you tried testing this scenario with the controller or the CLI?

Reply:
@DhariniJeeva @calcaide  yea authorized actions for dynamic host are only `no-op` and `read`
<img width="1044" alt="image" src="https://github.com/hashicorp/boundary-ui/assets/24277002/e48e5d64-4282-43e2-b36b-ff03f10366c1">

Reply:
@dvonanderson gotcha!!! then the current implementation is on point. Since we realize that host-set type is exclusive for what table show let me question the current authorized actions. I am glad we clarify that and the current implementation follows it.